src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Sql/Tables/TokenText.sql

```diff
@@ -4,7 +4,10 @@
     ResourceSurrogateId         bigint              NOT NULL,
     SearchParamId               smallint            NOT NULL,
     Text                        nvarchar(400)       COLLATE Latin1_General_CI_AI NOT NULL,
-    IsHistory                   bit                 NOT NULL
+    IsHistory                   bit                 NOT NULL,
+    CONSTRAINT PK_TokenText PRIMARY KEY NONCLUSTERED(ResourceTypeId, ResourceSurrogateId, SearchParamId, Text)
```

Comment:
>Text [](http://example.com/codeflow?start=105&length=4)

I have feeling what adding TextHash to this table would be beneficial as well.

It will cost you 32 bytes, but your primary key would have constant size of int + int+ int +binary(32) rather than
int+int+int+nvarchar(400).

If average size of text is more than 64 bytes, it would be better to have hash here as well.

Reply:
Agree! I will add TextHash column to this table